tests/SConscript

```diff
@@ -2,4 +2,5 @@
 import os
 from lsst.sconsUtils import scripts, env
 
-scripts.BasicSConscript.tests()
+# testCoadds.py must be entirely skipped, and Scons doesn't like that
+scripts.BasicSConscript.tests(ignoreList=['testCoadds.py'])
```

Comment:
I now see the skips in the test itself, which answers the question, but it's not obvious from reading the message. Perhaps something like {{All tests in testCoadds.py are skipped and Scons doesn't like that}}.

It's a pity about Scons not liking that; it'd be much safer to leave the skips in place